spec/enumerables_spec.rb

```diff
@@ -0,0 +1,129 @@
+require_relative '../enumerables.rb'
+
+describe Enumerable do
+  let(:array) { [1, 2, 3, 4, 5, 6, 7, 8, 9] }
+  let(:words_array) { %w[ant bear cat] }
+  let(:nil_true) { [nil, true, 99, false] }
+  let(:proced) { proc { |n| n * 2 } }
+  describe '#my_each' do
+    it 'Returns each element' do
+      expect(array.my_each { |n| n * n }).to eql(array)
+    end
+
+    it 'Returns enumerator' do
+      expect(array.my_each.class).to be(Enumerator)
+    end
+  end
+
+  describe '#my_each_with_index' do
+    it 'Returns each element' do
+      expect(array.my_each_with_index { |n| n * n }).to eql(array)
+    end
```

Comment:
Same as above, you can compare it with the actual method.

Reply:
Done! Thank you for pointing it out! We changed it as suggested